plugins/login-plugin/src/Utils/Customization/index.js

```diff
@@ -4,19 +4,25 @@ import { platformSelect } from "@applicaster/zapp-react-native-utils/reactUtils"
 import { populateConfigurationValues } from "@applicaster/zapp-react-native-utils/stylesUtils";
 import MESSAGES from "../../Components/AssetFlow/Config";
 
-const manifestJson = platformSelect({
-  ios: require("../../../manifests/ios_for_quickbrick.json"),
-  tvos: require("../../../manifests/tvos_for_quickbrick.json"),
-  android: require("../../../manifests/android.json"),
-  android_tv: require("../../../manifests/android_tv_for_quickbrick.json"),
-  web: require("../../../manifests/samsung_tv.json"),
-  samsung_tv: require("../../../manifests/samsung_tv.json"),
-  // lg_tv: require("../../../manifests/lg_tv.json"),
-  default: require("../../../manifests/android.json"),
-});
+const manifestJson = () => {
+  try {
+    return platformSelect({
+      ios: require("../../../manifests/ios_for_quickbrick.json"),
+      tvos: require("../../../manifests/tvos_for_quickbrick.json"),
+      android: require("../../../manifests/android.json"),
+      android_tv: require("../../../manifests/android_tv_for_quickbrick.json"),
+      web: require("../../../manifests/samsung_tv.json"),
+      samsung_tv: require("../../../manifests/samsung_tv.json"),
+      lg_tv: require("../../../manifests/lg_tv.json"),
+      default: require("../../../manifests/android.json"),
+    });
+  } catch (error) {
+    console.warn("Could not load manifest", error);
+  }
+};
 
 export function pluginIdentifier() {
-  return manifestJson.identifier;
+  return manifestJson().identifier;
```

Comment:
Do we want the app to crash if `manifestJson` returns undefined?
If so maybe it's better to throw from try-catch?

Reply:
You are right, the try catch is not actually handling it, do we want the app to fail or should we provide a default so that the app continues?

This is also a question I ask myself with all of the cell styles, they suffer from this same dilemma. 

For example:
```
Could not load manifest ReferenceError: platformSelect is not defined
    at manifestJson (/Users/sherwino/Desktop/deprecate/try.js:3:5)
    at pluginIdentifier (/Users/sherwino/Desktop/deprecate/try.js:19:10)
    at Object.<anonymous> (/Users/sherwino/Desktop/deprecate/try.js:22:32)
    at Module._compile (internal/modules/cjs/loader.js:999:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1027:10)
    at Module.load (internal/modules/cjs/loader.js:863:32)
    at Function.Module._load (internal/modules/cjs/loader.js:708:14)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:60:12)
    at internal/main/run_main_module.js:17:47
/Users/sherwino/Desktop/deprecate/try.js:19
  return manifestJson().identifier;
                       ^

TypeError: Cannot read property 'identifier' of undefined
    at pluginIdentifier (/Users/sherwino/Desktop/deprecate/try.js:19:24)
    at Object.<anonymous> (/Users/sherwino/Desktop/deprecate/try.js:22:32)
    at Module._compile (internal/modules/cjs/loader.js:999:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1027:10)
    at Module.load (internal/modules/cjs/loader.js:863:32)
    at Function.Module._load (internal/modules/cjs/loader.js:708:14)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:60:12)
    at internal/main/run_main_module.js:17:47
  ```

I guess letting it throw at tryCatch would be the expected behavior

Reply:
`platformSelect is not defined`? 

Yes agree, let's remove tryCatch then so we get a more explicit error. :+1:

Reply:
Ok now throwing.